### PR TITLE
Update jsonFields to jsonFieldMappings and LoggingMessages.nlsprops

### PR DESCRIPTION
**Pull request by Yushan-Lin**
#8813
#8814 
#build

### CODE REVIEW

File: dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops [hunk 1]
```diff
@@ -141,16 +141,16 @@ JSON_FIELDS_FORMAT_WARNING.explanation=The JSON logging field name mapping is in
 JSON_FIELDS_FORMAT_WARNING.useraction=Ensure the format of the JSON logging field name mapping property is correct. The JSON field name mapping can be specified in a number of ways, specified in the logging configuration documentation.
 
 JSON_FIELDS_FORMAT_WARNING_2=TRAS3008W: The JSON logging field name mapping property has too many or too few tokens in an entry.
-JSON_FIELDS_FORMAT_WARNING_2.explanation=The JSON logging field name mapping property should be formatted knownFieldName:newFieldName or eventType:knownFieldName:newFieldName.
-JSON_FIELDS_FORMAT_WARNING_2.useraction=The JSON logging field name mapping property should be formatted knownFieldName:newFieldName or eventType:knownFieldName:newFieldName.
+JSON_FIELDS_FORMAT_WARNING_2.explanation=The JSON logging field name mapping property should be formatted knownFieldName:newFieldName or source:knownFieldName:newFieldName.
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops]: should we change 'knownFieldName' to 'defaultFieldName' (applies to explanation and useraction)?

File: dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops [hunk 2]
```diff
@@ -141,16 +141,16 @@ JSON_FIELDS_FORMAT_WARNING.explanation=The JSON logging field name mapping is in
...
 
-JSON_FIELDS_NO_MATCH=TRAS3009W: Provided JSON logging field name does not match one of the known field names for the specified event(s)
+JSON_FIELDS_NO_MATCH=TRAS3009W: Provided JSON logging field name does not match one of the known field names for the specified source(s)
 JSON_FIELDS_NO_MATCH.explanation=Provided JSON logging field name does not match one of the known field names
 JSON_FIELDS_NO_MATCH.useraction=Please provide a JSON logging field name that matches one of the known fields (e.g. message, ibm_datetime, ibm_sequence)
 
-JSON_FIELDS_INCORRECT_EVENT_TYPE=TRAS3010W: Provided event type does not match one of the known event types
-JSON_FIELDS_INCORRECT_EVENT_TYPE.explanation=The accepted event types are: liberty_message, liberty_trace, liberty_accesslog, liberty_ffdc
-JSON_FIELDS_INCORRECT_EVENT_TYPE.useraction=The accepted event types are: liberty_message, liberty_trace, liberty_accesslog, liberty_ffdc
+JSON_FIELDS_INCORRECT_EVENT_TYPE=TRAS3010W: Provided JSON logging field mapping source identifier does not match one of the known sources
```
Comment (dmuelle) [inline dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops]: change to

The provided JSON logging field mapping source identifier...

File: dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops [hunk 3]
```diff
@@ -141,16 +141,16 @@ JSON_FIELDS_FORMAT_WARNING.explanation=The JSON logging field name mapping is in
 JSON_FIELDS_FORMAT_WARNING.useraction=Ensure the format of the JSON logging field name mapping property is correct. The JSON field name mapping can be specified in a number of ways, specified in the logging configuration documentation.
 
 JSON_FIELDS_FORMAT_WARNING_2=TRAS3008W: The JSON logging field name mapping property has too many or too few tokens in an entry.
-JSON_FIELDS_FORMAT_WARNING_2.explanation=The JSON logging field name mapping property should be formatted knownFieldName:newFieldName or eventType:knownFieldName:newFieldName.
-JSON_FIELDS_FORMAT_WARNING_2.useraction=The JSON logging field name mapping property should be formatted knownFieldName:newFieldName or eventType:knownFieldName:newFieldName.
+JSON_FIELDS_FORMAT_WARNING_2.explanation=The JSON logging field name mapping property should be formatted defaultFieldName:newFieldName or source:defaultFieldName:newFieldName.
```
Comment (dmuelle) [inline dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops]: change to

The JSON logging field name mapping property must be formatted

File: dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops [hunk 4]
```diff
@@ -141,16 +141,16 @@ JSON_FIELDS_FORMAT_WARNING.explanation=The JSON logging field name mapping is in
...
+JSON_FIELDS_FORMAT_WARNING_2.useraction=The JSON logging field name mapping property should be formatted defaultFieldName:newFieldName or source:defaultFieldName:newFieldName.
```
Comment (dmuelle) [inline dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops]: change to

The JSON logging field name mapping property must be formatted

**Comment by andrea-pichler** [comment]
Requested message review completed by L2

**Pull request merged**